distribution/docker/build.gradle

```diff
@@ -165,6 +165,41 @@ tasks.register("copyKeystore", Sync) {
   }
 }
 
+tasks.register("checkSecurityAuditLayoutPatternIdentical") {
+  def patternPropertyKey = "appender.audit_rolling.layout.pattern"
+  doLast {
+    def coreLog4jProperties = new Properties()
+    def originalLog4j = project(":x-pack:plugin:core").file('src/main/config/log4j2.properties')
+    originalLog4j.withInputStream { input ->
+      coreLog4jProperties.load(input)
+    }
+
+    if (false == coreLog4jProperties.containsKey(patternPropertyKey)) {
+      throw new GradleException("The [" + originalLog4j.getPath() + "] file changed such that the layout pattern is not " +
```

Comment:
This is Groovy, so need to do string concatination, just use groovy string interpolation (ex: `The [${originalLog4j.path}] file...`).